pyproject.toml

```diff
@@ -90,8 +90,8 @@ warn_unused_ignores = true
 # https://github.com/python/mypy/issues/4008
 exclude = "^tests/test_data/"
 
-[tool.mypy.overrides]
-modules = ["tests.*"]
+[[tool.mypy.overrides]]
+module = ["tests.*"]
```

Comment:
Fixed the error:
`toml config file contains a [[tool.mypy.overrides]] section, but no module to override was specified`